R/request.R

```diff
@@ -226,6 +226,9 @@ set_params <- function(path, route = NULL){
   nms <- c()
   pms <- list()
   for(i in seq_along(path_split)){
+    if(i > length(route$components))
+      next
```

Comment:
since we are not mutating `route$components`, shouldn't this be a `break` instead of `next`? because once the condition `i > length(route$components` is `TRUE`, it will be `TRUE` for all subsequent i's.